bpf-gpl/nat.h

```diff
@@ -45,25 +45,43 @@
 #define VXLAN_ENCAP_SIZE	(sizeof(struct ethhdr) + sizeof(struct iphdr) + \
 				sizeof(struct udphdr) + sizeof(struct vxlanhdr))
 
-// Map: NAT level one.  Dest IP and port -> ID and num backends.
+/* Prefix len = (dst_addr + port + protocol + src_addr) in bits. */
+#define NAT_PREFIX_LEN_WITH_SRC_MATCH 88
```

Comment:
could be defined using the sizeof as Shaun suggested to give the value a meaning? It still going to need some maintenance, but at least the calculation of the value is explained by the code.